src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -49,6 +49,10 @@
 
     <!-- Package source-build artifacts -->
     <RepositoryReference Include="package-source-build" />
+
+    <!-- Testing. -->
+    <RepositoryReference Include="scenario-tests" />
```

Comment:
Does this have the potential to cause issues when we introduce parallel builds?  Specifically will a dependency need to be declared on package-source-build to ensure this is built after the package-source-build project?  If this could be built in parallel or before the package project then the test's nupkg could end up in the source-built artifacts tarball.  Worst case is non-deterministic behavior.  I don't think we want these tests in the artifacts tarball.

Reply:
When introducing parallel builds, we only need to have dotnet.proj depend on the 'top-level' projects. I think it's that's package-source-build? Maybe scenario-tests too.. They all then depend on their dependencies. We enable parallel builds of dependencies and scenario tests could get built very early.

Part of the parallel build work will require that we create separate inputs for each component so that we don't end up with race conditions or unintentional dependencies. That would mean that PSB does not depend on scenario tests, and would not see the outputs of scenario tests as its inputs.

This is described in the SB issue.